pkg/cmd/dev/build.go

```diff
@@ -114,6 +114,18 @@ func (d *dev) build(cmd *cobra.Command, commandLine []string) error {
 	ctx := cmd.Context()
 	cross := mustGetFlagString(cmd, crossFlag)
 
+	bazelRcLine, err := d.setUpCache(ctx)
```

Comment:
In `doctor` we check the variable `DEV_NO_REMOTE_CACHE` and skip setting up the cache if it's set. Can you do the same here?

Reply:
Done.